public/js/main-template.js

```diff
@@ -12,93 +12,124 @@ $(document).ready(function () {
     asideBarRight = document.getElementById("AsideBarRight");
     fieldContainer = document.getElementById("field-container");
     installcontainer = document.getElementById("install-container");
+   if (document.getElementsByClassName("PannelIcon")[0]) {

```

Comment:
i would say give unique id's to each panel icons and this way will be easy and unique way to maintain instead of array[0] or array[1]